src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,40 @@
+.catalog {
+  display: grid;
+  grid-auto-columns: $card-width;
+  grid-auto-rows: $card-height;
+  grid-gap: 48px;
+  justify-items: start;
+  align-items: stretch;
```

```suggestion
  justify-content: center;
```
Let's center the elements